test/addons/argocd/start

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+from drenv import kubectl
+from drenv import commands
+from drenv import cache
+
+
+def deploy(cluster):
+    print("Deploying argocd")
+    kubectl.apply("--filename", "argocd-ns.yaml", context=cluster)
+    path = cache.path("addons/argocd-2.11-install.yaml")
+    cache.fetch(".", path)
+    kubectl.apply("--filename", path, "--namespace", "argocd", context=cluster)
+
+
+def wait(cluster):
+    print("Waiting until all deployments are available")
+    kubectl.wait(
+        "deploy",
+        "--all",
+        "--for=condition=Available",
+        "--namespace=argocd",
+        "--timeout=300s",
+        context=cluster,
+    )
+
+
+if len(sys.argv) != 4:
+    print(f"Usage: {sys.argv[0]} hub cluster1 cluster2")
+    sys.exit(1)
+
+os.chdir(os.path.dirname(__file__))
+hub = sys.argv[1]
+
+deploy(hub)
+wait(hub)
+
+# Need switch to argocd ns first, otherwise will hit argocd bug
+# see https://github.com/argoproj/argo-cd/issues/14167
+kubectl.config("use-context", hub)
+kubectl.config("set-context", "--current", "--namespace=argocd")
+
+commands.run("argocd", "login", "--core")
+
+for name in sys.argv[1:]:
+    try:
+        commands.run("argocd", "cluster", "add", name, "-y")
```

Comment:
This can fail since the cluster did not start yet. In the regional dr environment this failure is unlikely since we run after submariner addon, so all clusters are already running, but when testing the addon on a minimal env this fails randomly.

To wait for the clusters add code like this:


```python
from drenv import cluster

def wait_for_clusters(cluster1, cluster2):
    for c in cluster1, cluster2:
        print(f"Waiting until cluster '{c}' is ready")
        cluster.wait_until_ready(c)
```

Reply:
ok. added code to wait for clusters ready.